index.php

```diff
@@ -31,6 +31,11 @@
 $settingsResponse = json_decode($values);
 $values = $l->suggest('fossasia', 10, 'asc');
 $suggestResponse = json_decode($values);
+$values = $l->aggregations("fossasia", "", "", array("mentions","hashtags"), 10);
+// Some other possible inputs
+// $l->aggregations("fossasia", "2015-01-10", "2015-10-21", array("mentions","hashtags"), 10);
+// $l->aggregations("fossasia", "", "", array("mentions","hashtags"));
```

Comment:
I will add a check for single element. User shouldn't use array for a single element :+1: 
`implode` shall do the job. I don't know why I used `foreach`